src/xmipp/applications/tests/function_tests/test_volume_subtraction_main.cpp

```diff
@@ -0,0 +1,170 @@
+sph  + 1.   -1.412 0.000 -0.539 .10\n\
+sph  + 1.   -0.873 -0.873 -0.873 .10\n\
+sph  + 1.   -0.539 1.412 0.000 .10\n\
+sph  + 1.   0.539 1.412 0.000 .10\n\
+sph  + 1.   -0.539 -1.412 0.000 .10\n\
+sph  + 1.   0.539 -1.412 0.000 .10\n\
+sph  + 1.   0.000 0.000 1.618 .10\n\
+sph  + 1.   -0.500 -0.809 1.309 .10\n\
+sph  + 1.   0.500 -0.809 1.309 .10\n\
+sph  + 1.   0.500 0.809 1.309 .10\n\
+sph  + 1.   -0.500 0.809 1.309 .10\n\
+sph  + 1.   0.000 0.000 1.618 .10\n\
+sph  + 1.   0.500 -0.809 1.309 .10\n\
+sph  + 1.   0.809 -1.309 0.500 .10\n\
+sph  + 1.   1.309 -0.500 0.809 .10\n\
+sph  + 1.   1.309 0.500 0.809 .10\n\
+sph  + 1.   0.809 1.309 0.500 .10\n\
+sph  + 1.   0.500 0.809 1.309 .10\n\
+sph  + 1.   1.309 -0.500 0.809 .10\n\
+sph  + 1.   1.618 0.000 0.000 .10\n\
+sph  + 1.   1.309 0.500 0.809 .10\n\
+sph  + 1.   0.000 0.000 -1.618 .10\n\
+sph  + 1.   -0.500 0.809 -1.309 .10\n\
+sph  + 1.   0.500 0.809 -1.309 .10\n\
+sph  + 1.   0.500 0.809 -1.309 .10\n\
+sph  + 1.   0.809 1.309 -0.500 .10\n\
+sph  + 1.   1.309 0.500 -0.809 .10\n\
+sph  + 1.   0.500 -0.809 -1.309 .10\n\
+sph  + 1.   -0.500 -0.809 -1.309 .10\n\
+sph  + 1.   0.000 0.000 -1.618 .10\n\
+sph  + 1.   1.309 0.500 -0.809 .10\n\
+sph  + 1.   1.618 0.000 0.000 .10\n\
+sph  + 1.   1.309 -0.500 -0.809 .10\n\
+sph  + 1.   1.309 -0.500 -0.809 .10\n\
+sph  + 1.   0.809 -1.309 -0.500 .10\n\
+sph  + 1.   0.500 -0.809 -1.309 .10\n\
+sph  + 1.   -0.500 0.809 1.309 .10\n\
+sph  + 1.   -0.809 1.309 0.500 .10\n\
+sph  + 1.   -1.309 0.500 0.809 .10\n\
+sph  + 1.   -1.309 0.500 0.809 .10\n\
+sph  + 1.   -1.618 0.000 0.000 .10\n\
+sph  + 1.   -1.309 -0.500 0.809 .10\n\
+sph  + 1.   -1.309 -0.500 0.809 .10\n\
+sph  + 1.   -0.809 -1.309 0.500 .10\n\
+sph  + 1.   -0.500 -0.809 1.309 .10\n\
+sph  + 1.   -0.500 0.809 -1.309 .10\n\
+sph  + 1.   -0.809 1.309 -0.500 .10\n\
+sph  + 1.   -1.309 0.500 -0.809 .10\n\
+sph  + 1.   -1.309 0.500 -0.809 .10\n\
+sph  + 1.   -1.618 0.000 0.000 .10\n\
+sph  + 1.   -1.309 -0.500 -0.809 .10\n\
+sph  + 1.   -1.309 -0.500 -0.809 .10\n\
+sph  + 1.   -0.809 -1.309 -0.500 .10\n\
+sph  + 1.   -0.500 -0.809 -1.309 .10\n\
+sph  + 1.   0.000 1.618 0.000 .10\n\
+sph  + 1.   -0.809 1.309 -0.500 .10\n\
+sph  + 1.   -0.809 1.309 0.500 .10\n\
+sph  + 1.   0.809 1.309 0.500 .10\n\
+sph  + 1.   0.809 1.309 -0.500 .10\n\
+sph  + 1.   0.000 1.618 0.000 .10\n\
+sph  + 1.   0.000 -1.618 0.000 .10\n\
+sph  + 1.   -0.809 -1.309 -0.500 .10\n\
+sph  + 1.   -0.809 -1.309 0.500 .10\n\
+sph  + 1.   0.809 -1.309 0.500 .10\n\
+sph  + 1.   0.809 -1.309 -0.500 .10\n\
+sph  + 1.   0.000 -1.618 0.000 .10\n";
+
+
+class TransformWindowTest : public ::testing::Test
+{
+protected:
+    virtual void SetUp()
+    {
+        XMIPP_TRY
+        ;
+        XMIPP_CATCH
+    }
+};
+
+
+TEST_F( TransformWindowTest, unitcell)
+{
+    //create phantom volume
+    char filename[] = "/tmp/mytemp.XXXXXX"; // template for our file.
+    int fd = mkstemp(filename);    // Creates and opens a new temp file r/w.
```

Comment:
is `testing::TempDir()` available in our GTest?

Reply:
This was copy-pasted from other test just as a template to start. There is nothing of this in the test for this PR now.